internal/service/backup/find.go

```diff
@@ -6,9 +6,9 @@ package backup
 import (
 	"context"
 
-	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/backup"
 	awstypes "github.com/aws/aws-sdk-go-v2/service/backup/types"
+	"github.com/aws/aws-sdk-go/aws"
```

Comment:
I think there are some changes not rebased entirely correct, like this one:
```suggestion
	"github.com/aws/aws-sdk-go-v2/aws"
```